python/tk_desktop/project_model.py

```diff
@@ -207,6 +207,21 @@ def filterAcceptsRow(self, source_row, source_parent):
         """
         QSortFilterProxyModel override to base filtering on our cached values.
         """
+        if not hasattr(self, "_ids_in_order"):
+            """
+            The cache probably is not ready yet. With Qt5.15 sometimes we are
+            called by setSourceModel so for sure the cache is not  ready yet.
+
+            The best value to return here is TRUE which is the default in Qt.
+            Hopefully we will be called again once the cache is up-to-date and
```

Comment:
How can we validate that?

Reply:
In QT filterAcceptsRow returns True by default according to docs https://doc.qt.io/qt-5/qsortfilterproxymodel.html#filterAcceptsRow

We want to filter only when the cached values are ready.

Reply:
We confirmed that the method is called again debugging the code with breakpoints.